### DISCUSSION TITLE
❓ Where should I ask my question? ❓

### DISCUSSION THREAD

**Discussion by stichbury** [kedro-org/kedro#1987] [Idea]
Should I ask questions here or on Slack?

**Comment by stichbury** [comment]
We recommend you use the #questions channel of the [Kedro community Slack organisation](https://slack.kedro.org/) instead of here. It'll get you a faster response as more of the community are checking it regularly.

Thank you for being part of our community! [Here's a guide to where to post what](https://github.com/kedro-org/kedro/discussions/1986) 🖤

**Reply by aaronsgithub** [reply]
The slack channel doesn't appear to be open to the public at present.

If I click the link above and try to sign in, it says I don't have an account on this workspace.
If I click the slack link on kedro.org it says "This link is no longer active".

**Reply by yetudada** [reply]
Hey @aaronsgithub! Thanks for flagging that our link has expired (we're updating it right now), in the meanwhile you can use this link: ~~https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw~~ _(edit: expired)_

**Reply by astrojuanlu** [reply]
Invites expire from time to time, if https://slack.kedro.org doesn't work, please ping us here on GitHub

**Comment by astrojuanlu** [comment]
Closing in favour of https://github.com/kedro-org/kedro/discussions/3820
